src/prelude/internals.mo

```diff
@@ -406,3 +406,8 @@ func @create_actor_helper(wasm_module_ : Blob, arg_ : Blob) : async Principal =
   });
   return canister_id_;
 };
+
+// raw calls
+func @call_raw(p : Principal, m : Text, a : Blob) : async Blob {
```

Comment:
Why to define this in `prim.mo`?

Reply:
I tried, but our checker complains about await's in prim, so I just moved it here instead. Sneaky.